src/modules/sca/include/sca_policy_parser.hpp

```diff
@@ -0,0 +1,71 @@
+#pragma once
+
+#include <filesystem_wrapper.hpp>
+#include <sca_policy.hpp>
+
+#include <nlohmann/json.hpp>
+#include <string>
+#include <unordered_map>
+#include <yaml-cpp/yaml.h>
+
+/// @brief Compares two strings based on their length and then alphabetical order.
+struct StringLengthGreater
```

Comment:
Repeated, use the one from `sca_utils.hpp`.

Reply:
I've deleted `StringLengthGreater` and `PolicyVariables` from `sca_utils.hpp` because they are only used in `sca_policy_parser.hpp`.